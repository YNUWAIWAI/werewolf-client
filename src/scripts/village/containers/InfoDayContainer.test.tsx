import * as React from 'react'
import InfoDayContainer from './InfoDayContainer'
import IntlProviderContainer from './IntlProviderContainer'
import {MORNING} from '../constants/Phase'
import {Provider} from 'react-redux'
import fakeStore from './fakeStore'
import {mount} from 'enzyme'

test('<InfoDayContainer />', () => {
  const store = fakeStore(
    {
      base: {
        date: 1,
        phase: MORNING,
        phaseTimeLimit: 100
      }
    }
  )
  const wrapper = mount(
    <Provider store={store} >
      <IntlProviderContainer>
        <InfoDayContainer />
      </IntlProviderContainer>
    </Provider>
  )

  expect(wrapper.html()).toMatchSnapshot()
})
