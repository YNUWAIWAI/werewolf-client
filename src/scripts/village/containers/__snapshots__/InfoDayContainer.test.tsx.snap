// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoDayContainer /> 1`] = `"<div class=\\"info--day\\"><div class=\\"info--day--text\\"><span>Day 1: morning</span><span>00'00 left</span></div></div>"`;
